APIs/openEO/Python_Client/Python.qmd

```diff
@@ -0,0 +1,346 @@
+---
+title: "Getting started with Python client"
+format:
+  html:
+    code-fold: false
+    
+jupyter: python3
+---
+
+This Getting Started guide will give you just a simple overview of the capabilities of the openEO Python client library.
+More in-depth information can be found in its [official documentation](https://open-eo.github.io/openeo-python-client/){target="_blank"}.
+
+## Installation
+
+The openEO Python client library is available on [PyPI](https://pypi.org/project/openeo/){target="_blank"}
+and can easily be installed with a tool like `pip`, for example:
```

Comment:
should we consider adding conda install instructions for windows users? At least one user was still having trouble finding it on conda, I don't know why, but perhaps this is not 100% trivial for some reason?

Reply:
added terminal code for installing in conda env